<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_controllerDialog</class>
 <widget class="QDialog" name="dialog_null_controllerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_controllerDialog_controller">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>控制器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_controllerName">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>控制器名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_controllerOne">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_controllerTwo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_controllerThree">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_controllerFour">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_controllerDialog_controllerName_1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_controllerDialog_controllerName_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_controllerDialog_controllerName_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_controllerDialog_controllerName_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_controllerDialog_sure">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_controllerDialog_cancel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_controllerDialog_use">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>控制器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_controllerDialog_status">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_controllerDialog_controller_1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_controllerDialog_controller_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_controllerDialog_controller_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_controllerDialog_controller_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
